### PR TITLE
Added stop command to Taxi Worker

### PR DESCRIPTION
**Pull request by MichaelAiz**

### CODE REVIEW

File: modules/targetAcquisition/taxi/TaxiWorker.py [hunk 1]
```diff
@@ -9,9 +9,8 @@ def taxi_worker(pause, exitRequest, pipelineIn, pipelineOut):
         pause.release()
         
         frame = pipelineIn.get()
-        taxi.main(frame)
-
-        pipelineOut.put()
+        if taxi.main(frame).latestDistance == 0:
+            pipelineOut.put(taxi.main(frame))
```
Comment (Shrinjay) [inline modules/targetAcquisition/taxi/TaxiWorker.py:13]: Minor thing, don't call main over and over again. Call taxi.main(frame) and save it to a variable like ```command = taxi.main(frame)``` use that variable instead

File: modules/targetAcquisition/taxi/TaxiWorker.py [hunk 2]
```diff
@@ -9,9 +9,8 @@ def taxi_worker(pause, exitRequest, pipelineIn, pipelineOut):
         pause.release()
         
         frame = pipelineIn.get()
-        taxi.main(frame)
-
-        pipelineOut.put()
+        if taxi.main(frame).latestDistance == 0:
```
Comment (Shrinjay) [inline modules/targetAcquisition/taxi/TaxiWorker.py:12]: First check that the return from taxi.main has a property called latestDistance, use OR.

Comment (Xierumeng) [inline modules/targetAcquisition/taxi/TaxiWorker.py:12]: Data validation: check incomging and outgoing data.

File: modules/targetAcquisition/taxi/TaxiWorker.py [hunk 3]
```diff
@@ -9,9 +9,8 @@ def taxi_worker(pause, exitRequest, pipelineIn, pipelineOut):
         pause.release()
         
         frame = pipelineIn.get()
```
Comment (Shrinjay) [inline modules/targetAcquisition/taxi/TaxiWorker.py:11]: Validate that frame is not None, if it is, just continue

**Review by Shrinjay — CHANGES_REQUESTED** [review]

**Pull request closed without merging**